cypress/integration/a11y.spec.js

```diff
@@ -69,114 +93,68 @@ describe('Check accessiblity', () => {
-                    checkRadioAndClickContinue('Yes');
-                }
-            });
-        });
+        it('eApp eligibility questions', () => {
+            clickContinueBtn();
+            checkA11y('[Error] Which service would you like?');
 
-        context('eApp file upload', () => {
-            beforeEach(() => {
-                checkRadioAndClickContinue('e-Apostille service');
-            });
-
-            it('Get the documents legalised using the e-Apostille service', () => {
-                findByRole('link', {
-                    name: 'skip to the start of the service',
-                }).click();
-            });
-
-            it('Add your PDFs', () => {
-                passEappStartScreen();
-            });
-
-            it('Add your PDFs - 1 file uploaded', () => {
-                passEappStartScreen();
-                uploadTestFile();
-            });
-
-            it('Would you like to give this application a reference?', () => {
-                passEappStartScreen();
-                uploadTestFile();
-                clickContinueBtn();
-            });
-        });
+            findByLabelText('e-Apostille service').check();
+            checkA11y('Select radio option and check a11y');
 
-        context('eApp applications section', () => {
-            function selectFirstApplication() {
-                findAllByTestId('eApp-ref-link').first().click();
-            }
-
-            it('Your account', () => {
-                findByRole('link', {
-                    name: 'Applications',
-                }).click();
-            });
-
-            it('View app', () => {
-                findByRole('link', {
-                    name: 'Applications',
-                }).click();
-                selectFirstApplication();
-            });
-        });
+            checkRadioAndClickContinue('e-Apostille service');
+            checkA11y(
+                '1 - Is the e-Apostille accepted in the destination country?'
+            );
+
+            checkRadioAndClickContinue('Yes');
+            checkA11y(
+                '2 - Check if the documents are eligible for the e-Apostille service'
+            );
+
+            checkRadioAndClickContinue('Yes');
+            checkA11y(
+                '3 - Have the PDFs been notarised and digitally signed by a notary?'
+            );
+
+            checkRadioAndClickContinue('Yes');
+            checkA11y(
+                'Get the documents legalised using the e-Apostille service'
+            );
+
+            go('back');
+            checkRadioAndClickContinue('No');
+            checkA11y('You cannot use this service');
+
+            go('back');
+            checkRadioAndClickContinue('Yes');
+            clickContinueBtn();
+
+            visit('/upload-files');
+            checkA11y('eApp file upload');
+
+            uploadTestFile();
+            wait(500);
+            checkA11y('Add your PDFs - 1 file uploaded');
```

Comment:
is it necessary to wait? cypress waits by itself

Reply:
Ah didn't know cypress waits by itself thanks for pointing that out. I tried your suggestion in slack to use `findByText` but it didn't seem to work:

<img width="1732" alt="Screenshot 2022-02-24 at 16 39 17" src="https://user-images.githubusercontent.com/1377253/155579759-748cfdab-cbb6-46f1-bc77-7e0f75666625.png">

I think it's because that text is an .ejs conditional and doesn't show up on initial page load 🤷 

But `contains` seems to work better:

<img width="1594" alt="Screenshot 2022-02-24 at 17 50 25" src="https://user-images.githubusercontent.com/1377253/155579908-495d2a5b-61ba-4c93-b295-6ca9bc6cdd19.png">

Reply:
cypress runs on the browser, so it will just be rendered html. ejs is server side rendered as well, so ejs shouldn't interfere. I can't tell if it's the styling, but it looks like for findByText, this was entered ` 1 file was uploaded ` (whitespace at the start and end). you can use `findByText("thing I want to search", { exact: false })`. (but this is minor and contain works fine also)

Reply:
Ah you're right about the exact false flag because the text is actually on a new line in the code so maybe it's looking for exact whitespacing as well. I'll give that a go tomorrow and see if that fixes it.